src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  padding: 5px;
+  margin: auto;
+  gap: $gap;
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} {
+      & > .calendar__day:first-child {
+        margin-left: if(
+          $day == 'mon',
+          0,
+          calc(
+            (#{$day-size} * #{$index}) +
+              (#{$gap} * #{$index}) +
+              (#{$padding}) +
+              (#{$border-size}) *
```

Comment:
Ensure that the `$border-size` variable is defined. If not, this could lead to a compilation error or unexpected styling results.